unit4.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+// Unit 4. Objects and Functions
+
+// - Создать объект car, добавить к нему свойство color со значением 'черный'
+
+const car = {
+  color: 'черный',
+};
+
+// - Изменить свойство color объекта car на 'зеленый'
+
+car.color = 'зеленый';
+
+// - В объект car добавить свойство power, которое является функцией и выводит в консоль мощность двигателя
+
+car.power = function (hp) {
```

Comment:
Naming: `hp` is not good and obvious name for horsepower argument. It is better to rename it to `horsepower`.